content/desktop/install/archlinux.md

```diff
@@ -37,19 +33,29 @@ $ sudo pacman -S gnome-terminal
 2. [Install Docker client binary on Linux](../../engine/install/binaries.md#install-daemon-and-client-binaries-on-linux). On Arch-based distributions, users must install the Docker client binary.
    Static binaries for the Docker client are available for Linux (as `docker`).
 
-3. Download the Arch package from the [release](../release-notes.md) page.
+3. Download the latest Arch package from the [Release notes](../release-notes.md).
```

Comment:
Actually, with `pacman -U`, couldn't they just install the package from download.docker.com directly?